inst/extdata/plotstyle.csv

```diff
@@ -215,8 +215,8 @@ Trade|Primary Energy|Coal|Value;Coal;#999959;;
 Trade|Primary Energy|Gas|Value;Gas;#191999;;
 Trade|Primary Energy|Oil|Value;Oil;#337fff;;
 Trade|Uranium|Value;Uranium;#33ff00;;
-CAZ;;#e6194B;;
-CHA;;#3cb44b;;
+CAZ;Canada;#e6194B;;
+CHA;mainly China;#3cb44b;;
```

Comment:
Just to be sure, you did not mean "mainland China" here?

Reply:
No its China and some other countries